src/postw90/get_oper.F90

```diff
@@ -55,6 +55,15 @@ module w90_get_oper
   complex(kind=dp), allocatable, save :: SH_R(:, :, :, :) ! <0n|sigma_x,y,z.H|Rm>
   !! $$\langle 0n | \sigma_{x,y,z}.H  | Rm \rangle$$
 
+  !temporary
+  ! <0n|sigma_a. (r-R)_b|Rm>
+  !
+  complex(kind=dp), allocatable, save :: SAA_R(:, :, :, :, :)
```

Comment:
Change `!temporary` to e.g. `!Spin Hall using Ryoo method`? Also, the comments could be made more consistent with the previous lines.